doc/release-notes-31384.md

```diff
@@ -0,0 +1,25 @@
+- Node and Mining
+
+---
+
+- **PR #31384** fixed an issue where block reserved weight for coinbase transaction and headers
+ was done in two separate places. When the node constructs a new block, e.g. a `getblocktemplate`
+ RPC call doesn't generate the coinbase transaction. Instead, it reserves `4,000` weight units (WU)
+ for it and block header info. Before this pull request, the default for `-blockmaxweight` was `3,996,000`,
+ The mining code effectively added another `4,000` weight units (WU) to this reservation, leading to a template
+ size of `3,992,000`.
+
+- The fix consolidates the reservation into a single place and introduces a new startup option,
+  `-blockreservedweight` (default: `8,000` weight units). This option specifies the weight units
+ reserved for the coinbase transaction. The default value of `-blockreservedweight` was chosen
```

Comment:
ad1bc03245181b00a25ea0182373eddae1c151e1: the header and

Reply:
Done